### PR TITLE
Inserted docstring for two dimensional sor

### PR DESCRIPTION
**Pull request by zenoone**
I inserted the docstrings

### CODE REVIEW

**Comment by cwehmeyer** [comment]
meh, fix your typos!

**Review by cwehmeyer — CHANGES_REQUESTED** [review]
Please check your indentions and fix typos

**Comment by zenoone** [comment]
could you be a bit more specific please, 
how should indentions look like?
thanks

File: urbanbroccoli/sor.py [hunk 1]
```diff
@@ -31,13 +31,13 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
     return phi
 
 def sor_2d(rho, hx, hy, epsilon, maxiter, maxerr, w):
-    """Performs a sucsesive over relaxation for the poisson equation of a two dimensional discretized charge distribution:
+    """Performs a sucsessive over-relaxation for the Poisson equation of a two dimensional discretized charge distribution:
```
Comment (cwehmeyer) [inline urbanbroccoli/sor.py]: This line has the correct indention but is too long. Use a very short description and, if necessary, a longer text block after two line breaks.

File: urbanbroccoli/sor.py [hunk 2]
```diff
@@ -31,13 +31,13 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
...
           Parameters:
```
Comment (cwehmeyer) [inline urbanbroccoli/sor.py]: This line has an indention level of 10 spaces. Indentions should be multiples of four and, most important, they should be consistent. Use an indention level of four spaces, here.

File: urbanbroccoli/sor.py [hunk 3]
```diff
@@ -31,13 +31,13 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
...
             rho (ndarray)
```
Comment (cwehmeyer) [inline urbanbroccoli/sor.py]: Lines 36-49: remove four spaces indention.

**Review by cwehmeyer — CHANGES_REQUESTED** [review]
Please look at the three exemplary notes and make sure that the entire docstring fits that scheme

File: urbanbroccoli/sor.py [hunk 4]
```diff
@@ -31,6 +31,23 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
     return phi
 
 def sor_2d(rho, hx, hy, epsilon, maxiter, maxerr, w):
+    """Performs a 2D sucsessive over-realxation.
+    
+    Performs a sucsessive over-relaxation for the Poisson equation to determine the 
+    potential resulting from a two dimensional discretized charge distribution.
+        
+    Parameters:
```
Comment (cwehmeyer) [inline urbanbroccoli/sor.py:39]: This is what I had in mind.

**Review by cwehmeyer — APPROVED** [review]

**Pull request merged**